Marlin/temperature.cpp

```diff
@@ -1544,70 +1546,79 @@ void Temperature::isr() {
   #endif
 
   #if DISABLED(SLOW_PWM_HEATERS)
```

Comment:
Have you tried it?
after the preprocessor run the code reads as follows:
    constexpr uint8_t pwm_mask = 1 ? 1 : 0;
which is both syntactically and semantically correct.